test/runtests.jl

```diff
@@ -3,6 +3,7 @@ using Test
 using Flux, LegolasFlux
 using LegolasFlux: Weights, FlatArray, ModelRow
 using Arrow
+using Random
```

Comment:
Is it possible we want to use StableRNG here (and then seeded rands, below)?

Reply:
good idea, done!